src/bot.py

```diff
@@ -47,7 +50,41 @@ async def search_send(ctx, *args):
     else:
         await ctx.send("No issues found")
 
+
+@bot.command(name='/svn help', description="Retrieve information about the available commands and their usage")
+async def show_help(ctx, *args):
+    """Display a list of available commands and get information on how to use the Savanna Bot"""
+
+    if '/svn help' in args:
+        await ctx.send("""
+            Description: An example program to demonstrate how to organize help in the command-line interface.
+                        
+            #Search by language
+                        
+            With the language qualifier you can search for issues and pull requests within repositories that are written in a certain language.
+            
+            Qualifier:
+            language:LANGUAGE   Example: language:ruby state:open matches open issues that are in Ruby repositories.
```

Comment:
Eu acho que o exemplo tem que incluir o comando do bot, tipo `/svn language:ruby state:open` matches open issues that are in Ruby repositories.